src/libcpu/src/jit/jit_float.cpp

```diff
@@ -340,75 +340,56 @@ frsp(PPCEmuAssembler& a, Instruction instr)
    return true;
 }
 
-template <bool ShouldNegate>
+template <bool ShouldAbs, bool ShouldNegate>
 static bool
-fabsGeneric(PPCEmuAssembler& a, Instruction instr)
+fmrGeneric(PPCEmuAssembler& a, Instruction instr)
```

Comment:
Could you point out which of these registers is the one which is actually corrupting ps1?  They should all operate soely on the low double of the XMM.